src/BuoyancyModels/BuoyancyModels.jl

```diff
@@ -52,10 +52,10 @@ include("g_dot_b.jl")
 include("buoyancy_field.jl")
 
 Base.show(io::IO, b::SeawaterBuoyancy{FT}) where FT =
-    println(io, "SeawaterBuoyancy{$FT}: g = $(b.gravitational_acceleration)", '\n',
+    print(io, "SeawaterBuoyancy{$FT}: g = $(b.gravitational_acceleration)", '\n',
                 "└── equation of state: $(b.equation_of_state)")
 
 Base.show(io::IO, eos::LinearEquationOfState{FT}) where FT =
-    println(io, "LinearEquationOfState{$FT}: ", @sprintf("α = %.2e, β = %.2e", eos.α, eos.β))
+    print(io, "LinearEquationOfState{$FT}: ", @sprintf("α = %.2e, β = %.2e", eos.α, eos.β))
```

Comment:
Can you clarify what this change does?

Reply:
`println` adds a newline after printing the string, I think.

Reply:
yes, that's it... and it was creating this extra lines -->

<img width="966" alt="Screen Shot 2021-11-13 at 11 21 31 am" src="https://user-images.githubusercontent.com/7112768/141597950-4dc63649-bd57-4f69-9b36-dd888bfb4799.png">